### DISCUSSION TITLE
README.md Front, Back e Root

### DISCUSSION THREAD

**Discussion by mikaelmonteirodev** [EasyRoomCompany/Guardian#57] [Ideas]
Para a documentação ficar organizada existe a ideia de incluir no README do projeto, links que direcionam para READMEs secundários, para o Frontend e para o Backend. Assim talvez fique mais organizado.

> - **README Principal**: Contendo todas as especificações de uso da aplicação.
     - README Frontend: Especificações técnicas do Front;
     - README Backend: Especificações técnicas do Back.

O que acham?!

**Comment by kevinDsousa** [comment]
Acho que seja uma ideia interessante poderiamos aderir

**Reply by ivannaprata** [reply]
Concordo! Podemos aderir.

**Reply by gabrielsousarod** [reply]
gostei demais!!!!

**Comment by mikaelmonteirodev** [comment]
Ok! Será feito então.
